### PR TITLE
feat: enable channel creation for logged-in user only

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.atlassian.net/browse/CLNP-251

Enabled the channel creation button if there's only the logged in user && users.length === 0.

### CODE REVIEW

File: src/modules/CreateChannel/components/InviteUsers/index.tsx [hunk 1]
```diff
@@ -103,28 +105,27 @@ const InviteUsers: React.FC<InviteUsersProps> = ({
           });
           return;
         }
-        if (selectedUserList.length > 0) {
```
Comment (AhyoungRyu) [inline src/modules/CreateChannel/components/InviteUsers/index.tsx:106]: Nothing changed in this if block, but just removed the condition cause there'll be no case this `selectedUserList.length === 0` with [this logic](https://github.com/sendbird/sendbird-uikit-react/pull/665/files#diff-08c2d4af32c32fa2cabfaf2e14278e24dea1f367e411a2483e84f38bdb73790cR97)

File: src/modules/CreateChannel/components/InviteUsers/index.tsx [hunk 2]
```diff
@@ -91,10 +91,12 @@ const InviteUsers: React.FC<InviteUsersProps> = ({
       titleText={titleText}
       submitText={submitText}
       type={ButtonTypes.PRIMARY}
-      disabled={Object.keys(selectedUsers).length === 0}
+      disabled={users.length > 0 && Object.keys(selectedUsers).length === 0}
```
Comment (sravan-s) [inline src/modules/CreateChannel/components/InviteUsers/index.tsx:94]: If possible, can you add a  test for this -> 
`should enable create button when there are no users in the list` 
`should disable create button when there are users on list but none's checked` 

Separate PR is okay

Else, comments are fine too

**Review by sravan-s — APPROVED** [review]

File: src/modules/CreateChannel/components/InviteUsers/index.tsx (hunk 2 above)
Comment (sravan-s) [reply src/modules/CreateChannel/components/InviteUsers/index.tsx:94]: Because this is a bit unobvious part ~

Comment (HoonBaek) [inline src/modules/CreateChannel/components/InviteUsers/index.tsx:94]: This logic stil disable the submit button when there's no selected user

**Review by HoonBaek — APPROVED** [review]

File: src/modules/CreateChannel/components/InviteUsers/index.tsx (hunk 2 above)
Comment (AhyoungRyu) [reply src/modules/CreateChannel/components/InviteUsers/index.tsx:94]: Right. This part is related to this conversation https://sendbird.slack.com/archives/C050Z6DA70A/p1688089591024089?thread_ts=1688086902.570209&cid=C050Z6DA70A if I got your point correctly

Comment (AhyoungRyu) [reply src/modules/CreateChannel/components/InviteUsers/index.tsx:96]: https://github.com/sendbird/sendbird-uikit-react/pull/665/commits/16b47590799c020590999022f86408a7a652aa9b added a comment first. Will add some unit tests separately!

**Pull request merged**